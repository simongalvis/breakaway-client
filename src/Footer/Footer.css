footer{
   
    height: 25px;
    background-color: black;
    color: white;
    padding: 4% 0;
    
}
a{
    color:white
}
p{
    margin:0;
    
}