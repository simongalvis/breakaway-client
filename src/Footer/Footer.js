import React from 'react';
import '../Footer/Footer.css';



class Footer extends React.Component{



    render(){
        return(
            <footer>
                <p>Â© - <a href='http://simongalvis.com/'>simongalvis.com</a></p>
            </footer>
        )
    }
}

export default Footer;