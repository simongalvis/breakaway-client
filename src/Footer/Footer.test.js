import React from 'react'
import ReactDOM from 'react-dom'
import Footer from './Footer'
import renderer from 'react-test-renderer';

describe(`Footer component`, () =>{
  it(`renders without crashing`, () =>{
    const div = document.createElement('div');
  
    ReactDOM.render(<Footer/>, div);
  
    ReactDOM.unmountComponentAtNode(div);
  })
  it(`Renders UI as expected`, () =>{
    const tree = renderer
    .create(<Footer/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  })
})
