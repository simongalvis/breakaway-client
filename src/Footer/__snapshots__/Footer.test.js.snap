// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Renders UI as expected 1`] = `
<footer>
  <p>
    Â© - 
    <a
      href="http://simongalvis.com/"
    >
      simongalvis.com
    </a>
  </p>
</footer>
`;
